client/src/component/AuthorTimelines/index.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { Tabs, Tab } from 'react-bootstrap';
+import PropTypes from 'prop-types';
+import TimelineList from '../TimelineList';
+import './AuthorTimelines.css';
+
+const AuthorTimelines = (props) => (
+  <>
+    {props.isLoggedIn ? (
+      <Tabs defaultActiveKey="public" id="public-private-tabs">
+        <Tab eventKey="public" title="Public">
+          <div className="TimelineListContainer">
+            <TimelineList timelines={props.publicTimelines} />
+          </div>
+        </Tab>
+        <Tab eventKey="private" title="Private">
+          <div className="TimelineListContainer">
+            <TimelineList timelines={props.privateTimelines} />
+          </div>
+        </Tab>
+      </Tabs>
+    ) : (
+      <TimelineList timelines={props.publicTimelines} />
+    )}
+  </>
```

Comment:
no need for wrapper i guess. since both if or else wll return one enclosing tab.

Reply:
without Fragments <> </> compiler fails to recognize expression in { } and throws a parsing error